rules_gapic/gapic.bzl

```diff
@@ -56,19 +57,21 @@ gapic_srcjar = rule(
             allow_single_file = True,
             mandatory = True,
         ),
-        "gapic_yaml": attr.label(mandatory = True, allow_single_file = True),
-        "artifact_type": attr.string(mandatory = False, default = "GAPIC_CODE"),
-        "language": attr.string(mandatory = True),
+        "artifact_type": attr.string(mandatory = False), #default = "GAPIC_CODE"
```

Comment:
Why was the default removed for this now optional option?

Reply:
Because it makes sense only for the monolithic generator and can break (as unexpected argument) microgenerators.